core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -497,14 +497,19 @@ private void checkTimestamp() throws VerificationException {
                     TimeUtils.dateTimeFormat(allowedTime), allowedTime.toEpochMilli()));
     }
 
+    /**
+     * Sums up all SigOps in all transactions of this block.
+     *
+     * @return sum of SigOps
+     */
+    public int sigOpCount() {
+        return transactions.stream().mapToInt(Transaction::getSigOpCount).sum();
```

Comment:
Needs null check!

Reply:
Added, although it would throw NPE immediately at the beginning of the existing statement.